app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -73,12 +75,18 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => {
         // For testing - users will be redirected if the analysis is unavailable
         // before reaching this return, but tests reach this return and
         // analysis.overall is expected to be an object (EXP-800)
-        if (analysisUnavailable(analysis)) return;
+        if (!analysis || analysisUnavailable(analysis)) return;
```

Comment:
I took `!analysis` out of the `analysisUnavailable` check and placed it here to silence Typescript down the entire page (I was able to remove a lot of `?`s and `!` where we know `analysis` can't be undefined due to this check). `analysisAvailable` checks if it's undefined by way of `analysis?.whatever` chaining so it shouldn't change anything functionally.